components/toggle_modal_button.jsx

```diff
@@ -58,7 +58,7 @@ export default class ModalToggleButton extends React.Component {
                 {...props}
                 className={'style--none ' + props.className}
                 onClick={clickHandler}
-                data-testid='add-group'
```

Comment:
this had to be changed since previously every instance of toggle modal button on a page had the same test id